.github/workflows/integration-build-product.yml

```diff
@@ -774,8 +798,424 @@ jobs:
           docker system prune -f
           docker image prune --all -f
 
+  Validation-Simulator-Tests:
+    needs: [Build-Spawner, Build-Simulator]
+    runs-on: integration-pipeline-simulation
+    steps:
+      - uses: rtCamp/action-cleanup@master
+        if: ${{ inputs.with_simulation_tests }}
+      - name: Checkout
+        if: ${{ inputs.with_simulation_tests }}
+        uses: actions/checkout@v3
+
+      - name: Agent info
+        run: |
+          echo "public ip: $(curl ipinfo.io/ip)"
+          echo "private ip: $(hostname -I | awk '{print $1}')"
+
+      - name: Install CI Scripts
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: python3 -m pip install integration-pipeline==$CI_INTEGRATION_SCRIPTS_VERSION --ignore-installed
+
+      - name: unstash raised_meta
+        if: ${{ inputs.with_simulation_tests }}
+        uses: actions/download-artifact@v3
+        with:
+          name: raised_meta
+          path: .
+
+      - name: unstash manifest
+        if: ${{ inputs.with_simulation_tests }}
+        uses: actions/download-artifact@v3
+        with:
+          name: manifest
+          path: .
+
+      - name: unstash sim_configs
+        if: ${{ inputs.with_simulation_tests }}
```

Comment:
Why not put this on the job instead of all steps?

Reply:
When you skip a job, you can not have it in the critical path.

Reply:
The next jobs will also skip

Reply:
Good to know, thanks <3